office/libreoffice/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           boost 1.0
 
 name                libreoffice
 version             7.2.3.1
-revision            0
+revision            1
```

Comment:
I don't think this qualifies as a revision bump, since we're not changing the contents of the port itself. The [docs](https://guide.macports.org/#reference.keywords/) also suggest not to rev bump to fix build failures.

The only reason I know this is since I've been corrected on this myself :) https://github.com/macports/macports-ports/commit/65512ee30539ec36aaae353035929eda248f67d1

Reply:
Yes let's undo this revbump. No need.

Reply:
@harens I wasn't sure should I update libreoffice revision or not. Without it, it won't trigger rebuild of libreoffice => user still needs to rebuild it by themself (and it requires huge time).

if it is ok for @Tatsh I'd love to keep revision bump here, to force buildbot's to rebuild it for users.

Reply:
From https://github.com/macports/macports-ports/commit/65512ee30539ec36aaae353035929eda248f67d1:

> exactly, for fixing build failure one shouldn't increase the revision. Ideally, the build should just be started on the buildbot, but for that you need an account. I think that eventually it will retry failed builds automatically, so it will get done at some point.

A revision bump would force everyone to rebuild this, including machines where it was working fine beforehand. My understanding is that some authorised users i.e. not me ;) can get the arm buildbots to rebuild the binary without a revision bump.

Is that ok @catap? Thanks again for your work on this, greatly appreciated.

Reply:
@harens let make things faster and simpler when